src/main/webapp/pages/createPlaceGuide_initPage.js

```diff
@@ -25,8 +25,13 @@ function initPage() {
       mapWidget.addGeolocationFunctionality();
       mapWidget.addLocationChoosingAndSavingFunctionality();
       map = mapWidget.map;
-      placeGuideManager = new PlaceGuideManager(
-          PlaceGuideManager.PAGE.CREATE_PLACE_GUIDE, map);
+      const placeGuideRepository =
+          new PlaceGuideRepository(
+              PlaceGuideRepository.QUERY_TYPE.CREATED_ALL_IN_MAP_AREA);
+      placeGuideManager = new PlaceGuideManager(PlaceGuideManager.PAGE.CREATE_PLACE_GUIDE, map);
+      google.maps.event.addListener(map, 'idle', function() {
+        placeGuideManager.refreshPlaceGuides(map.getBounds(), map.getZoom(), false);
+      });
```

Comment:
The listener to the map's idle state was move dto PlaceGuideManager. And it also handles initialising the PlaceGuideRepository.
Why did you move these back to initPage()?